source/features/clean-repo-sidebar.tsx

```diff
@@ -20,7 +20,10 @@ async function cleanLicenseText(): Promise<void> {
 }
 
 async function cleanReleases(): Promise<void> {
-	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]', {waitForChildren: false});
+	// Intent: select "Releases" section in right sidebar.
+	// But if "repository details" - "Website" has a link end with "/releases", it will be wrongly selected.
+	// @see refined-github/issues/4424 for more details.
+	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]:not([role])', {waitForChildren: false});
```

Comment:
```suggestion
	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]:not([role])', {waitForChildren: false}); // :not('[role]') excludes website links that end with "/releases" #4424
```

I did not test the code, however I think the above comment is enough.